Skype/UCWA/myContacts_ref.md

```diff
@@ -1,3 +1,8 @@
+---
+title: myContacts
```

Comment:
I would make this a tad more descriptive. Maybe "Learn about myContacts, which is a collection of contact resources."